include/libopencm3/efm32/lg/acmp.h

```diff
@@ -48,28 +48,28 @@
 #define ACMP_CTRL_WARMTIME_MASK		(0x7 << ACMP_CTRL_WARMTIME_SHIFT)
 #define ACMP_CTRL_WARMTIME(v)		\
 	(((v) << ACMP_CTRL_WARMTIME_SHIFT) & ACMP_CTRL_WARMTIME_MASK)
-#define ACMP_CTRL_WARMTIME_4CYCLES	ACMP_CTRL_WARMTIME(0)
-#define ACMP_CTRL_WARMTIME_8CYCLES	ACMP_CTRL_WARMTIME(1)
-#define ACMP_CTRL_WARMTIME_16CYCLES	ACMP_CTRL_WARMTIME(2)
-#define ACMP_CTRL_WARMTIME_32CYCLES	ACMP_CTRL_WARMTIME(3)
-#define ACMP_CTRL_WARMTIME_64CYCLES	ACMP_CTRL_WARMTIME(4)
-#define ACMP_CTRL_WARMTIME_128CYCLES	ACMP_CTRL_WARMTIME(5)
-#define ACMP_CTRL_WARMTIME_256CYCLES	ACMP_CTRL_WARMTIME(6)
-#define ACMP_CTRL_WARMTIME_512CYCLES	ACMP_CTRL_WARMTIME(7)
+#define ACMP_CTRL_WARMTIME_4CYCLES	(0)
+#define ACMP_CTRL_WARMTIME_8CYCLES	(1)
+#define ACMP_CTRL_WARMTIME_16CYCLES	(2)
+#define ACMP_CTRL_WARMTIME_32CYCLES	(3)
+#define ACMP_CTRL_WARMTIME_64CYCLES	(4)
+#define ACMP_CTRL_WARMTIME_128CYCLES	(5)
+#define ACMP_CTRL_WARMTIME_256CYCLES	(6)
+#define ACMP_CTRL_WARMTIME_512CYCLES	(7)
```

Comment:
When you are #defining simply a number do you really need parenthesis around it?